config/deploy/production.rb

```diff
@@ -15,6 +15,9 @@
   before :finished, 'newrelic:notice_deployment'
 end
 
+set :slack_webhook, 'https://hooks.slack.com/services/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'
```

Comment:
I'm not sure, do we think this is worth worrying about? it might be better to wait until its a problem